test_unstructured/partition/test_auto.py

```diff
@@ -78,7 +78,10 @@ def mock_docx_document():
     # NOTE(robinson) - this should get dropped because it is empty
     document.add_paragraph("", style="Normal")
     # NOTE(robinson) - this should get picked up as a narrative text
-    document.add_paragraph("This is my first thought. This is my second thought.", style="Normal")
+    document.add_paragraph(
```

Comment:
Why do all of these test files have formatting changes? That should only occur if there's a version changing in the linters or something. Please revert these.

Reply:
Can you your lint/format again or revert this file to what it is currently on main?